src/server/Document/component.jsx

```diff
@@ -73,6 +73,15 @@ const Document = ({
           />
         )}
         {scriptsAllowed && scriptTags}
+        {scriptsAllowed && (
+          <script
+            type="text/javascript"
+            /* eslint-disable-next-line react/no-danger */
+            dangerouslySetInnerHTML={{
```

Comment:
Isn't it bad to use this because of potential cross-site scripting?

Reply:
@HarveyPeachey that's correct. We had [an issue](https://github.com/bbc/simorgh/issues/2622) to sanitize all usage of `dangerouslySetInnerHTML` and [the consensus](https://github.com/bbc/simorgh/issues/2622#issuecomment-539897078) was to use the eslint rule to opt out especially in cases where there's no alternative

Reply:
I (think) there is an alternative. We have access to the html tag in this file so why not change the class of it after mount with useEffect by setting some state? Or something along those lines

Reply:
@jroebu14 @HarveyPeachey @jamesdonoh  seeing some weird behavior ever since I made the change from `className` to `class` [here](https://github.com/bbc/simorgh/pull/4810/files#r353040729) it works fine without the contents of the `dSih`, changing it back to className brings in the need for the `dSih` contents. not sure why that's happening